<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ThermoFunWidget</class>
 <widget class="QMainWindow" name="ThermoFunWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>872</width>
    <height>641</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ThermoFun Tabulation Widget</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="mainSplitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="opaqueResize">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="keyWidget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QSplitter" name="keySplitter">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QWidget" name="widget" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0">
            <property name="spacing">
             <number>1</number>
            </property>
            <property name="leftMargin">
             <number>1</number>
            </property>
            <property name="topMargin">
             <number>1</number>
            </property>
            <property name="rightMargin">
             <number>1</number>
            </property>
            <property name="bottomMargin">
             <number>1</number>
            </property>
            <item>
             <widget class="QTextEdit" name="edgeQuery">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="listsWidget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QSplitter" name="lstSplitter">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QWidget" name="inWidget" native="true">
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="2" colspan="2">
             <widget class="QComboBox" name="typeBox"/>
            </item>
            <item row="1" column="2" colspan="4">
             <widget class="QLineEdit" name="pName"/>
            </item>
            <item row="5" column="0" colspan="2">
             <widget class="QCheckBox" name="FormatBox">
              <property name="text">
               <string>Fixed Format</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="0" colspan="2" alignment="Qt::AlignLeft">
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Task Name</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="5" column="3">
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>Solvent Symbol</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_7">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="maximumSize">
               <size>
                <width>120</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>P</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="3" column="5">
             <widget class="QSpinBox" name="tPrecision">
              <property name="toolTip">
               <string extracomment="Output precision / digits after decimal point (in Fixed Format)"/>
              </property>
              <property name="toolTipDuration">
               <number>5</number>
              </property>
              <property name="value">
               <number>2</number>
              </property>
             </widget>
            </item>
            <item row="0" column="0" colspan="2" alignment="Qt::AlignLeft">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Schema</string>
              </property>
             </widget>
            </item>
            <item row="3" column="2">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Units</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="6" column="0" colspan="8">
             <widget class="QLabel" name="label_9">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Properties table</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>T</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="4" column="4">
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>Precision</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="3" column="4">
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>Precision</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="0" column="5">
             <widget class="QComboBox" name="pSourceTDB">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="3" column="3">
             <widget class="QComboBox" name="pTunit">
              <property name="currentText">
               <string>C Celsius</string>
              </property>
              <item>
               <property name="text">
                <string>C Celsius</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>K Kelvin</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QDoubleSpinBox" name="pTVal">
              <property name="minimum">
               <double>-500.000000000000000</double>
              </property>
              <property name="maximum">
               <double>500.000000000000000</double>
              </property>
              <property name="value">
               <double>25.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="0" column="4">
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>SourceTDB</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="2" column="0" colspan="6">
             <widget class="QLineEdit" name="pComment"/>
            </item>
            <item row="4" column="2">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Units</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QDoubleSpinBox" name="pPVal">
              <property name="maximum">
               <double>1000.000000000000000</double>
              </property>
              <property name="value">
               <double>1.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="4" column="3">
             <widget class="QComboBox" name="pPunit">
              <item>
               <property name="text">
                <string>b bar</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>B kbar</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>p Pa</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>P MPa</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>A atm</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="4" column="5">
             <widget class="QSpinBox" name="pPrecision">
              <property name="toolTip">
               <string extracomment="Output precision / digits after decimal point (in Fixed Format)"/>
              </property>
              <property name="toolTipDuration">
               <number>5</number>
              </property>
              <property name="value">
               <number>2</number>
              </property>
             </widget>
            </item>
            <item row="5" column="4" colspan="2">
             <widget class="QComboBox" name="pSolventSymbol">
              <property name="enabled">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="outWidget" native="true">
           <layout class="QGridLayout" name="gridLayout_3">
            <property name="leftMargin">
             <number>2</number>
            </property>
            <property name="topMargin">
             <number>2</number>
            </property>
            <property name="rightMargin">
             <number>2</number>
            </property>
            <property name="bottomMargin">
             <number>2</number>
            </property>
            <property name="spacing">
             <number>2</number>
            </property>
            <item row="1" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Temperature T points</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0" rowspan="2">
             <layout class="QGridLayout" name="statusLayout">
              <item row="0" column="0" alignment="Qt::AlignLeft">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Status</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="calcStatus">
                <property name="minimumSize">
                 <size>
                  <width>125</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>872</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDataBase">
    <property name="title">
     <string>&amp;Database</string>
    </property>
    <addaction name="actionSearch"/>
    <addaction name="actionSearch_Results"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionNew_Reset_Task_Definition"/>
    <addaction name="separator"/>
    <addaction name="actionExport_CfgFile"/>
    <addaction name="actionImport_CfgFile"/>
    <addaction name="separator"/>
    <addaction name="actionE_xit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionContents"/>
    <addaction name="separator"/>
    <addaction name="actionVersion"/>
    <addaction name="actionAuthors"/>
    <addaction name="actionLicense"/>
    <addaction name="separator"/>
    <addaction name="action_Help_About"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionSelect_Elements"/>
    <addaction name="separator"/>
    <addaction name="actionChange_Property_list"/>
    <addaction name="separator"/>
    <addaction name="actionReset_TP"/>
    <addaction name="actionRealloc_TP"/>
    <addaction name="separator"/>
    <addaction name="action_Set_Elemets_to_reactions"/>
    <addaction name="action_Set_Elemets_to_reactionsets"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_Calc">
    <property name="title">
     <string>&amp;Calculate</string>
    </property>
    <addaction name="actionCalculate_Properties"/>
    <addaction name="actionCalculate_TP"/>
   </widget>
   <widget class="QMenu" name="menuResults">
    <property name="title">
     <string>&amp;Results</string>
    </property>
    <addaction name="actionShow_Results"/>
   </widget>
   <widget class="QMenu" name="menuPreferences">
    <property name="title">
     <string>&amp;Preferences</string>
    </property>
    <addaction name="actionCalculate_Substances_Records_from_Dependent_Reactions"/>
    <addaction name="actionCalculate_Reactions_Records_from_Reactants"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menuPreferences"/>
   <addaction name="menu_Calc"/>
   <addaction name="menuResults"/>
   <addaction name="menuDataBase"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew_Reset_Task_Definition"/>
   <addaction name="actionExport_CfgFile"/>
   <addaction name="actionImport_CfgFile"/>
   <addaction name="separator"/>
   <addaction name="actionSelect_Elements"/>
   <addaction name="actionReset_TP"/>
   <addaction name="actionChange_Property_list"/>
   <addaction name="actionCalculate_Properties"/>
   <addaction name="separator"/>
   <addaction name="actionShow_Results"/>
  </widget>
  <widget class="QToolBar" name="nameToolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSearch"/>
   <addaction name="separator"/>
   <addaction name="action_Help_About"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionE_xit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionImport_CfgFile">
   <property name="icon">
    <iconset resource="../ThermoMatch.qrc">
     <normaloff>:/main/GUI/Icons/ShowFilesIcon24.png</normaloff>:/main/GUI/Icons/ShowFilesIcon24.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open Task Definition from File ...</string>
   </property>
   <property name="iconText">
    <string>Read Task Definition from File ...</string>
   </property>
   <property name="toolTip">
    <string>Read Task Definition from File ...</string>
   </property>
  </action>
  <action name="actionExport_CfgFile">
   <property name="icon">
    <iconset resource="../ThermoMatch.qrc">
     <normaloff>:/main/GUI/Icons/SaveCurrentRecordIcon24.png</normaloff>:/main/GUI/Icons/SaveCurrentRecordIcon24.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save Task Definition to File ...</string>
   </property>
   <property name="iconText">
    <string>Save Task Definition to File ...</string>
   </property>
   <property name="toolTip">
    <string>Save Task Definition to File...</string>
   </property>
  </action>
  <action name="action_Help_About">
   <property name="icon">
    <iconset resource="../ThermoMatch.qrc">
     <normaloff>:/main/GUI/Icons/QuestionMessageIcon24.png</normaloff>:/main/GUI/Icons/QuestionMessageIcon24.png</iconset>
   </property>
   <property name="text">
    <string> Help &amp;About</string>
   </property>
   <property name="toolTip">
    <string>Show Help VIewer ...</string>
   </property>
  </action>
  <action name="actionAuthors">
   <property name="text">
    <string>Au&amp;thors ...</string>
   </property>
  </action>
  <action name="actionLicense">
   <property name="text">
    <string>&amp;License ...</string>
   </property>
  </action>
  <action name="actionContents">
   <property name="text">
    <string>Help &amp;Contents ...</string>
   </property>
  </action>
  <action name="actionSearch">
   <property name="icon">
    <iconset resource="../ThermoMatch.qrc">
     <normaloff>:/main/GUI/Icons/SelectDisplayRecordIcon24.png</normaloff>:/main/GUI/Icons/SelectDisplayRecordIcon24.png</iconset>
   </property>
   <property name="text">
    <string>Create a &amp;Query to FInd Substances ...</string>
   </property>
  </action>
  <action name="actionSearch_Results">
   <property name="text">
    <string>E&amp;xplore this Query Results Table ...</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="actionVersion">
   <property name="text">
    <string>&amp;Version ...</string>
   </property>
  </action>
  <action name="actionCalculate_Properties">
   <property name="icon">
    <iconset resource="../ThermoMatch.qrc">
     <normaloff>:/main/GUI/Icons/CalculateRecordDataIcon24.png</normaloff>:/main/GUI/Icons/CalculateRecordDataIcon24.png</iconset>
   </property>
   <property name="text">
    <string>Calculate &amp;Properties</string>
   </property>
  </action>
  <action name="actionCalculate_TP">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../ThermoMatch.qrc">
     <normaloff>:/main/GUI/Icons/RTparmModuleIcon24.png</normaloff>:/main/GUI/Icons/RTparmModuleIcon24.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Tabulate the temperature and pressure points</string>
   </property>
  </action>
  <action name="actionShow_Results">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../ThermoMatch.qrc">
     <normaloff>:/main/GUI/Icons/ShowGEMResultsIcon24.png</normaloff>:/main/GUI/Icons/ShowGEMResultsIcon24.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Show Table with Results in CSV Widget ...</string>
   </property>
  </action>
  <action name="actionChange_Property_list">
   <property name="icon">
    <iconset resource="../ThermoMatch.qrc">
     <normaloff>:/main/GUI/Icons/DocumentsPileIcon24.png</normaloff>:/main/GUI/Icons/DocumentsPileIcon24.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Change Property List ...</string>
   </property>
  </action>
  <action name="actionRealloc_TP">
   <property name="text">
    <string>&amp;Realloc TP list ...</string>
   </property>
   <property name="toolTip">
    <string>Realloc TP pairs list</string>
   </property>
  </action>
  <action name="actionReset_TP">
   <property name="icon">
    <iconset resource="../ThermoMatch.qrc">
     <normaloff>:/main/GUI/Icons/SelectTP24.png</normaloff>:/main/GUI/Icons/SelectTP24.png</iconset>
   </property>
   <property name="text">
    <string>R&amp;eset TP list ...</string>
   </property>
   <property name="toolTip">
    <string>Reset TP list</string>
   </property>
  </action>
  <action name="actionNew_Reset_Task_Definition">
   <property name="icon">
    <iconset resource="../ThermoMatch.qrc">
     <normaloff>:/main/GUI/Icons/NewCreateRecordIcon24.png</normaloff>:/main/GUI/Icons/NewCreateRecordIcon24.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New/Reset Task Definition</string>
   </property>
  </action>
  <action name="actionSelect_Elements">
   <property name="icon">
    <iconset resource="../ThermoMatch.qrc">
     <normaloff>:/vertex/GUI/Icons/ICompModuleIcon.png</normaloff>:/vertex/GUI/Icons/ICompModuleIcon.png</iconset>
   </property>
   <property name="text">
    <string>Se&amp;lect Elements ...</string>
   </property>
  </action>
  <action name="action_Set_Elemets_to_reactions">
   <property name="text">
    <string> &amp;Set Elemets to reactions</string>
   </property>
  </action>
  <action name="actionCalculate_Substances_Records_from_Dependent_Reactions">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Calculate &amp;Substances Records from Dependent Reactions</string>
   </property>
  </action>
  <action name="actionCalculate_Reactions_Records_from_Reactants">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Calculate &amp;Reactions Records from Reactants</string>
   </property>
  </action>
  <action name="action_Set_Elemets_to_reactionsets">
   <property name="text">
    <string> &amp;Set Elemets to reactionsets</string>
   </property>
   <property name="toolTip">
    <string>Set Elemets to ReactionSets</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../ThermoMatch.qrc"/>
 </resources>
 <connections/>
</ui>
